main.go

```diff
@@ -18,7 +18,9 @@ package main
 
 import (
 	"flag"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
```

Comment:
can you move this down in to the external dependency list

Reply:
fixed..